packages/material-ui/src/MenuList/MenuList.d.ts

```diff
@@ -11,6 +11,19 @@ export interface MenuListProps extends StandardProps<ListProps, MenuListClassKey
 
 export type MenuListClassKey = ListClassKey;
 
+/**
+ * A permanently displayed menu following <https://www.w3.org/TR/wai-aria-practices/#menubutton>.
```

Comment:
?
```suggestion
 * A permanently displayed menu following https://www.w3.org/TR/wai-aria-practices/#menubutton.
```

Reply:
This is just some stylistic changes remark applies. It is valid markdown:

> URLs and URLs in angle brackets will automatically get turned into links.

-- https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

> The link URL may, optionally, be surrounded by angle brackets:

-- https://golem.ph.utexas.edu/~distler/maruku/markdown_syntax.html#link